R/importTaxpasta.R

```diff
@@ -0,0 +1,134 @@
+#' Import taxpasta-specific BIOM results to `TreeSummarizedExperiment`
+#'
+#' @param filename A character vector with one element that denotes the file
+#'   path to a BIOM file.
+#'
+#' @return A [TreeSummarizedExperiment::TreeSummarizedExperiment()] object.
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' tse <- importTaxpasta("profiles.biom")
+#' }
+#'
+#' @importFrom rhdf5 h5read
+#' @importFrom SummarizedExperiment rowData
+#' @importFrom SingleCellExperiment altExps
+importTaxpasta <- function(filename) {
```

Comment:
The parameter name can be just "file" as here: https://github.com/microbiome/mia/blob/devel/R/importHumann.R

Reply:
-> harmonize between functions

Reply:
That will shadow the `base::file` built-in function, but no problem.